src/Evolution/Systems/CurvedScalarWave/Worldtube/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY Worldtube)
```

Comment:
Oh maybe name this lib something like `CswWorldtube` or `ScalarWaveWorldtube`, because you want another GH worldtube lib later?

Reply:
^ agreed, and to distinguish it from CCE more, since CCE also uses worldtubes!